### ISSUE TITLE
Can't build with LTO set to true

### ISSUE THREAD

**Issue by osleg** [henriklaxhuber/perspektiv#1]
SUBJ
OS: Archlinux
Rust version 
```
stable-x86_64-unknown-linux-gnu (default)
rustc 1.28.0 (9634041f0 2018-07-30)
```

Error:
```
 Running `rustc --crate-name perspektiv src/main.rs --crate-type bin --emit=dep-info,link -C opt-level=z -C panic=abort -C lto -C codegen-units=4 --cfg 'feature="alsa"' --cfg 'feature="alsa_volume"' --cfg 'feature="default"' --cfg 'feature="x11"' --cfg 'feature="x11_backlight"' -C metadata=63a1440bb8eb7fbd -C extra-filename=-63a1440bb8eb7fbd --out-dir /home/alex/tmp/perspektiv/target/release/deps -C incremental=/home/alex/tmp/perspektiv/target/release/incremental -L dependency=/home/alex/tmp/perspektiv/target/release/deps --extern alsa=/home/alex/tmp/perspektiv/target/release/deps/libalsa-f180c2daf4eacc62.rlib --extern dirs=/home/alex/tmp/perspektiv/target/release/deps/libdirs-685b9e613ddbb3ec.rlib --extern gdk=/home/alex/tmp/perspektiv/target/release/deps/libgdk-499957ec96da7351.rlib --extern gdk_sys=/home/alex/tmp/perspektiv/target/release/deps/libgdk_sys-74bbe911dd232a89.rlib --extern glib=/home/alex/tmp/perspektiv/target/release/deps/libglib-60b98ef1001f2945.rlib --extern gtk=/home/alex/tmp/perspektiv/target/release/deps/libgtk-46c8ee15d9d49165.rlib --extern lazy_static=/home/alex/tmp/perspektiv/target/release/deps/liblazy_static-89947ffdd1d206d6.rlib --extern libc=/home/alex/tmp/perspektiv/target/release/deps/liblibc-aa5ce6c303853dae.rlib --extern log=/home/alex/tmp/perspektiv/target/release/deps/liblog-e02e68e600b28877.rlib --extern serde=/home/alex/tmp/perspektiv/target/release/deps/libserde-392ff05e6fb4527c.rlib --extern serde_derive=/home/alex/tmp/perspektiv/target/release/deps/libserde_derive-854f74af0f7d7f18.so --extern stderrlog=/home/alex/tmp/perspektiv/target/release/deps/libstderrlog-46b2baa3c0dae7af.rlib --extern threlm=/home/alex/tmp/perspektiv/target/release/deps/libthrelm-cd4b30194b0d3560.rlib --extern toml=/home/alex/tmp/perspektiv/target/release/deps/libtoml-5283b92d1d5275fa.rlib --extern x11=/home/alex/tmp/perspektiv/target/release/deps/libx11-8c4db35bf43e43fc.rlib -L native=/usr/lib -L native=/usr/lib -L native=/usr/lib -L native=/usr/lib`
error: can't perform LTO when compiling incrementally

error: Could not compile `perspektiv`.

Caused by:
  process didn't exit successfully: `rustc --crate-name perspektiv src/main.rs --crate-type bin --emit=dep-info,link -C opt-level=z -C panic=abort -C lto -C codegen-units=4 --cfg feature="alsa" --cfg feature="alsa_volume" --cfg feature="default" --cfg feature="x11" --cfg feature="x11_backlight" -C metadata=63a1440bb8eb7fbd -C extra-filename=-63a1440bb8eb7fbd --out-dir /home/alex/tmp/perspektiv/target/release/deps -C incremental=/home/alex/tmp/perspektiv/target/release/incremental -L dependency=/home/alex/tmp/perspektiv/target/release/deps --extern alsa=/home/alex/tmp/perspektiv/target/release/deps/libalsa-f180c2daf4eacc62.rlib --extern dirs=/home/alex/tmp/perspektiv/target/release/deps/libdirs-685b9e613ddbb3ec.rlib --extern gdk=/home/alex/tmp/perspektiv/target/release/deps/libgdk-499957ec96da7351.rlib --extern gdk_sys=/home/alex/tmp/perspektiv/target/release/deps/libgdk_sys-74bbe911dd232a89.rlib --extern glib=/home/alex/tmp/perspektiv/target/release/deps/libglib-60b98ef1001f2945.rlib --extern gtk=/home/alex/tmp/perspektiv/target/release/deps/libgtk-46c8ee15d9d49165.rlib --extern lazy_static=/home/alex/tmp/perspektiv/target/release/deps/liblazy_static-89947ffdd1d206d6.rlib --extern libc=/home/alex/tmp/perspektiv/target/release/deps/liblibc-aa5ce6c303853dae.rlib --extern log=/home/alex/tmp/perspektiv/target/release/deps/liblog-e02e68e600b28877.rlib --extern serde=/home/alex/tmp/perspektiv/target/release/deps/libserde-392ff05e6fb4527c.rlib --extern serde_derive=/home/alex/tmp/perspektiv/target/release/deps/libserde_derive-854f74af0f7d7f18.so --extern stderrlog=/home/alex/tmp/perspektiv/target/release/deps/libstderrlog-46b2baa3c0dae7af.rlib --extern threlm=/home/alex/tmp/perspektiv/target/release/deps/libthrelm-cd4b30194b0d3560.rlib --extern toml=/home/alex/tmp/perspektiv/target/release/deps/libtoml-5283b92d1d5275fa.rlib --extern x11=/home/alex/tmp/perspektiv/target/release/deps/libx11-8c4db35bf43e43fc.rlib -L native=/usr/lib -L native=/usr/lib -L native=/usr/lib -L native=/usr/lib` (exit code: 101)
```

**Comment by kartik-bhagatwala** [comment]
Getting the same error here, also on Arch.

**Comment by he-la** [comment]
Thanks. Should be fixed in a6316a4
